packages/graphql/src/services/sessions.ts

```diff
@@ -23,6 +142,20 @@ export async function startSession(
     return null
   }
 
+  try {
+    const results = await ctx.redisExec
```

Comment:
redixExec -> the redis in-memory cache that will store all of the results of a session while it is running (so no DB persistence during a live session, everything is done in-memory). Everything is stored under a "key" in redis, we use something like "s:SESSION\_ID:i:INSTANCE\_ID:results" to store the incoming responses on a specific question, for example.